src/api/emit.js

```diff
@@ -25,5 +30,8 @@ export default function (elem, name, opts = {}) {
   if (opts.cancelable === undefined) {
     opts.cancelable = true;
   }
+  if (opts.composed === undefined) {
```

Comment:
Definitely a breaking change, since they bubble outside of the "shadow dom" in the polyfill world. I suppose there should be a polyfill change to stop bubbling outside of the shadow dom at some point too.